python/datasources/acs_population.py

```diff
@@ -272,17 +310,25 @@ def write_to_bq(self, dataset, gcs_bucket):
         by_sex_standard_age_uhc = None
         by_sex_decade_plus_5_age_uhc = None
         by_sex_voter_age_uhc = None
+        by_sex_age_bjs = None
+        by_sex_age_juv_adult = None
+
         if not self.county_level:
             by_sex_standard_age_uhc = self.get_by_sex_age(
                 frames[self.get_table_name_by_sex_age_race()], get_uhc_standard_age_bucket)
             by_sex_decade_plus_5_age_uhc = self.get_by_sex_age(
                 frames[self.get_table_name_by_sex_age_race()], get_uhc_decade_plus_5_age_bucket)
             by_sex_voter_age_uhc = self.get_by_sex_age(
                 frames[self.get_table_name_by_sex_age_race()], get_uhc_voter_age_bucket)
+            by_sex_age_bjs = self.get_by_sex_age(
+                frames[self.get_table_name_by_sex_age_race()], get_bjs_age_bucket)
```

Comment:
@joshzarrabi , any thoughts on how to get `get_bjs_age_bucket` to only run on national and not on state? I guess it doesn't hurt to have the extra buckets available at the state level, but they aren't needed for incarceration and take extra time to compute. I only see the flag above for `if not self.county_level` and didn't see any other class properties for like `self.national_level`

Reply:
@benhammondmusic Here https://github.com/SatcherInstitute/health-equity-tracker/blob/0aa7406e2f2320e5a2d5b4fa0b7762999f2e03ad/python/datasources/acs_population.py#L290-L295, we automatically generate national level datasets based on the state ones. We do this because the lowest level the acs data goes to is state, so we need to generate the national based off of it. I would just leave it in the state dataset.

Reply:
ahh gotcha. thanks!